packages/mwp-app-server/src/util/getPlugins.js

```diff
@@ -17,20 +17,23 @@ import apiProxyPlugin from 'mwp-api-proxy-plugin';
  * @module ServerPlugins
  */
 
+const CSRF_COOKIE_NAME = 'x-mwp-csrf';
```

Comment:
followup will be to move this CSRF plugin code into a standalone plugin instead of managing it inside `mwp-app-server`